aspnetcore/blazor/host-and-deploy/app-base-path.md

```diff
@@ -60,19 +60,22 @@ For the sources of links that pertain to Blazor in ASP.NET Core apps:
 
 If you're rendering a Blazor app from different documents (for example, `/Admin/B/C/` and `/Admin/D/E/`), you must take the app base path into account, or the base path is different when the app renders in each document and the resources are fetched from the wrong URLs.
 
-There are two approaches to deal with the challenge of resolving relative links correctly:
+Choose one of the following strategies to keep relative links resolving correctly:
 
-* Map the resources dynamically using the document they were rendered on as the root.
-* Set a consistent base path for the document and map the resources under that base path.
+* Map resources dynamically using the document they were rendered on as the root. This requires custom routing logic (for example, <xref:Microsoft.AspNetCore.Routing.IDynamicEndpointMetadata> or a <xref:Microsoft.AspNetCore.Routing.MatcherPolicy>) and is typically reserved for highly dynamic sites.
+:::moniker range=">= aspnetcore-11.0"
 
-The first option is more complicated and isn't the most typical approach, as it makes navigation different for each document. Consider the following example for rendering a page `/Something/Else`:
+* Establish a consistent base path for the document and map all app resources beneath that path. Blazor Server/Auto apps can rely on the `BasePath` component to render `<base href>` automatically, while standalone WebAssembly apps (or scenarios that always use a fixed value) configure the literal `<base>` tag in the host page.
 
-* Rendered under `/Admin/B/C/`, the page is rendered with a path of `/Admin/B/C/Something/Else`.
-* Rendered under `/Admin/D/E/`, the page is rendered ***at the same path*** of `/Admin/B/C/Something/Else`.
+:::moniker-end
+
+:::moniker range="< aspnetcore-11.0"
 
-Under the first approach, routing offers <xref:Microsoft.AspNetCore.Routing.IDynamicEndpointMetadata> and <xref:Microsoft.AspNetCore.Routing.MatcherPolicy>, which in combination can be the basis for implementing a completely dynamic solution that determines at runtime about how requests are routed.
+* Establish a consistent base path for the document and map all app resources beneath that path. Configure the literal `<base>` tag in the host page for both Blazor Server and WebAssembly apps (or scenarios that always use a fixed value).
```

Comment:
"Blazor Server" is a bit vague per the naming conventions at >=8.0 after BWAs came out. Articles almost always adopt "server-side Blazor" language in these cases to cover both Blazor Server and BWAs. How about? ...

```suggestion
* Establish a consistent base path for the document and map all app resources beneath that path. Configure the literal `<base>` tag in the host page for both server-side Blazor apps and Blazor WebAssembly apps (or scenarios that always use a fixed value).
```